specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1770,6 +1872,136 @@
                 "sku"
             ]
         },
+        "TopologyList": {
+            "type": "object",
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/TopologyResource"
+                    }
+                },
+                "nextLink": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The URI to fetch the next page."
+                }
+            }
+        },
+        "TopologyResource": {
+            "type": "object",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/TopologyResourceProperties"
+                }
+            },
+            "required": [
+                "properties"
+            ],
+            "allOf": [{
+                    "$ref": "#/definitions/Resource"
+                },
+                {
+                    "$ref": "#/definitions/Location"
+                }
+            ]
+        },
+        "TopologyResourceProperties": {
+            "type": "object",
+            "properties": {
+                "calculatedDateTime": {
+                    "type": "string",
+                    "description": "The UTC time on which topology where calculated"
+                },
+                "Resources": {
+                    "type": "array",
+                    "description": "Azure resources which re part of this topology resource",
+                    "items": {
+                        "$ref": "#/definitions/TopologySingleResource"
+                    }
+                }
+            },
+            "required": [
+                "calculatedDateTime",
+                "Resources"
+            ]
+        },
+        "TopologySingleResource": {
+            "type": "object",
+            "properties": {
+                "resourceId": {
+                    "type": "string",
+                    "description": "Azure resource id"
+                },
+                "severity": {
+                    "type": "string",
+                    "description": "The security severity of the resource"
+                },
+                "recommendationExist": {
+                    "type": "boolean",
+                    "description": "Indicates if the resource has security recommendations"
+                },
+                "internetFacing": {
+                    "type": "boolean",
+                    "description": "Indicates if the resource has direct connectivity to the Internet"
+                },
+                "secureScore": {

```

Comment:
is this aligned with the secureScore that we are about to publish?
if not, please remove this property

Reply:
talked to PMs,
this field name should be changed